build_pyavroc.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2015 Byhiras (Europe) Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+cd $(dirname "$0")
```

Comment:
let's add comment with a usage example, because it's hard to understand meaning of `$0` and `$1`